a/antlr4/antlr4_v4.11.1_ubi_8.5.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package	: antlr4
+# Version	: v4.11.1
+# Source repo	: https://github.com/antlr/antlr4
+# Tested on	: UBI 8.5
+# Language      : Java, Python, C#, C++, Go, Swift
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Muskaan Sheik <Muskaan.Sheik@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=antlr4
+PACKAGE_VERSION=v4.11.1
```

Comment:
@muskaan2199  - can you please make change to accept PACKAGE_VERSION as param.
like PACKAGE_VERSION=${1:-v4.11.1}